server/test/services/applicant/ApplicantDataTest.java

```diff
@@ -63,4 +63,13 @@ public void withFailedUpdates() {
     assertThat(data.getFailedUpdates()).isEqualTo(ImmutableMap.of(samplePath, "invalid_value"));
     assertThrows(IllegalStateException.class, () -> data.asJsonString());
   }
+
+  @Test
```

Comment:
+1 blocking please add a browser test

Reply:
I will be honest with you- I am very poor in ts and last time it took me a long time to get the broken browser tests working. Unless I can pair program, it will be hard for me do it alone. I have done my best in adding Unit testing and also testing my branch (switch between main and my current branch to make sure the experience is seamless)

Reply:
I'm happy to pair program. I can do tomorrow 10:30-12pm. I'm experienced with TS and somewhat with civiform browsertests. So we should be able to figure out with two of us.

Reply:
Thanks @nbeloglazov :) I will send you an invite!